src/ASM/Test/ASMSquare.h

```diff
@@ -48,6 +54,8 @@ class ASMmxSquare : public ASMs2Dmx
     std::stringstream geo(ASMSquare::square);
     this->read(geo);
   }
+
```

Comment:
One blank line here is sufficient (coding standard)

Reply:
sure